lib/report/general_statistics.rb

```diff
@@ -34,8 +99,6 @@ def full_report(users, logins)
       }
     end
 
-  protected
```

Comment:
Why remove protected here?

Reply:
I moved it up

Reply:
https://github.com/alphagov/govuk-account-manager-prototype/pull/586/files#r555890249